libmenu/matemenu-tree.c

```diff
@@ -73,7 +73,7 @@ struct _MateMenuTree
   char *non_prefixed_basename;
   char *path;
   char *canonical_path;
-    GPtrArray  *collection_applet;
+ GPtrArray  *collection_applet;
```

Comment:
Indent

Reply:
@lukefromdc 
This intend issue wasn't fixed :/
See, how it looks in browser (raw formated)
https://github.com/mate-desktop/mate-menus/pull/86.diff